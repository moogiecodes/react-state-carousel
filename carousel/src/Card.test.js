import React from "react";
import { render } from "@testing-library/react";
import Card from "./Card";

// Smoke
it("renders without crashing", function () {
  render(<Card/>);
});

// Snapshot
it("matches the snapshot", function() {
  const { asFragment } = render(<Card currNum={1} />);
  expect(asFragment()).toMatchSnapshot();
});
