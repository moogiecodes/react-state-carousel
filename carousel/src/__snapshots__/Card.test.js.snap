// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    />
    <img
      class="Card-image"
    />
    <small
      class="Card-small"
    >
      Image 1 of .
    </small>
  </div>
</DocumentFragment>
`;
